assets/js/index.js

```diff
@@ -0,0 +1,262 @@
+    };
+  };
+
+  return forecastData.daily.slice(1, 8).map(callBack);
+};
+
+//get Data from API
+const getWeatherData = async (cityName) => {
+  const currentDataUrl = `https://api.openweathermap.org/data/2.5/weather?q=${cityName}&appid=${API_KEY}`;
+
+  const currentDataResponse = await fetch(currentDataUrl);
+
+  if (currentDataResponse.status === 200) {
+    const currentData = await currentDataResponse.json();
+
+    setCitiesInLs(cityName);
+
+    const lat = currentData.coord.lat;
+    const lon = currentData.coord.lon;
+    const name = currentData.name;
+
+    const forecastDataUrl = `https://api.openweathermap.org/data/2.5/onecall?lat=${lat}&lon=${lon}&appid=${API_KEY}&units=imperial`;
+
+    const forecastDataResponse = await fetch(forecastDataUrl);
+    const forecastData = await forecastDataResponse.json();
+
+    const current = getCurrentData(name, forecastData);
+
+    const forecast = getForecastData(forecastData);
+
+    return {
+      current: current,
+      forecast: forecast,
+    };
+  }
+};
+
+const getCitiesFromLs = () => {
+  //get cities from LS
+  const cities = JSON.parse(localStorage.getItem("recentCities")) ?? [];
+  return cities;
+};
+
+const setCitiesInLs = (cityName) => {
+  //get cities from LS
+  const cities = JSON.parse(localStorage.getItem("recentCities")) ?? [];
+
+  //if city does not exist
+  if (!cities.includes(cityName)) {
+    //insert cityName in cities
+    cities.push(cityName);
+
+    //set cities in LS
+    localStorage.setItem("recentCities", JSON.stringify(cities));
+  }
+};
+
+const getUVIClassName = (uvi) => {
+  //get uvi class name
+  if (uvi >= 0 && uvi < 3) {
+    return "uvi-blue";
+  } else if (uvi >= 3 && uvi < 5) {
+    return "uvi-yellow";
+  } else if (uvi >= 5 && uvi < 8) {
+    return "uvi-orange";
+  } else {
+    return "uvi-red";
+  }
+};
+
+//Construct Alert Message
+const renderAlertCard = () => {
+  const currentAlertCard = `<p>Please enter a valid city name</p>`;
+
+  dashboardContainer.append(currentAlertCard);
+};
+
+//construct current day weather card
+const renderCurrentWeatherCard = (current) => {
+  const currentWeatherCard = ` <div class="current-city-container">
+  <h2>
+    ${current.name} ${current.date}
+  </h2>
+  <div class="current-city-body">
+      <div class="current-city-content">
+        <p>Temp: ${current.temperature}&deg;F</p>
+        <p>Feels Like: ${current.feels_like}&deg;F</p>
+        <p>Wind: ${current.wind} MPH</p>
+        <p>Humidity: ${current.humidity}%</p>
+        <p>
+          UV index: <span class=${getUVIClassName(current.uvi)}>${
+    current.uvi
+  }</span>
+        </p>
+      </div>
+      <div class="current-city-img">
+        <img
+        src="https://openweathermap.org/img/w/${current.iconCode}.png"
+      />
+      </div>
+  </div>
+</div>`;
+
+  dashboardContainer.append(currentWeatherCard);
+};
+
+//construct forecast cards
+const renderForecastWeatherCards = (forecastData) => {
+  constructForecastCard = (each) => {
```

Comment:
Arrow functions should always be declared with `const`, don't forget!

Reply:
Thanks!!! Fixed!!